src/test/java/P1.java

```diff
@@ -0,0 +1,24 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class P1 extends BaseTest {
+    @Test
+    public void openAlbums () {
+        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+        actions = new Actions(driver);
+
+        login("india.messam@testpro.io", "slcTalgy");
+
+
+    }
+    //select albums tab
+    public void selectAlbumTab()  {
+        By.cssSelector("[href='#!/albums']");
+
+    }
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
 For example
    
![Clean_Test](https://github.com/Test-Pro-Bootcamp/A59-QA-Automation-Framework/assets/104761482/f935665b-f56d-4eee-9a8c-dacf6a866be3)